README.markdown

```diff
@@ -67,18 +67,18 @@ We encourage contributions from anybody and try to keep a warm and friendly comm
     </tr>
 </table>
 
-Internet connectivity is required for downloading Terasology via the Launcher, afterwards offline play is possible.
+Internet connectivity is required for downloading Terasology via the Launcher, afterwards playing offline is possible.
 
 For easy game setup (recommended) you can use our launcher - [download it here.](https://github.com/MovingBlocks/TerasologyLauncher/releases)
 
 For more information about playing, like hot keys or server hosting, see the [dedicated page](docs/Playing.md) or check out the [modules.](docs/Modules.md)
 
 
-#### Alternate Install Method
+#### Alternative Installation Methods
 
-If you know you already have a Java Runtime Environment installed, you may use a direct download release as an alternative to using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
+If you know you already have a Java Runtime Environment(JRE) installed, you may use a direct download release as an alternative for using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
```

Comment:
Taking that whitespace from above and adding it here :wink: so overall the whitespace count was good :grin: :+1: 
```suggestion
If you know you already have a Java Runtime Environment (JRE) installed, you may use a direct download release as an alternative for using the [launcher](https://github.com/MovingBlocks/TerasologyLauncher/releases). Java versions 8 and 11 are supported.
```